ui/src/app/views/error-handler/component.html

```diff
@@ -17,3 +17,22 @@ <h3 class="modal-title">This account has been disabled</h3>
     Please contact a system administrator to inquire about the status of your account.
   </div>
 </clr-modal>
+<clr-modal [(clrModalOpen)]="this.statusCheckService.firecloudDown">
```

Comment:
In this case, I think it probably makes sense to combine all the statusCheck modals into one. Agreed on that. I think that Error display needs a serious rewrite, but there's been enough stuff to do for the security review that I've punted a bit, and left those notifications as just an explanation that something is going wrong.

In this case, I actually do want to set it to false when the user closes the modal, so that if they close it and then refresh, they continue to get notified of the status? I probably want a bit of design input on how to handle errors before finalizing any of these though.